myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -52,77 +54,83 @@ <h4 class="card-badge-label">Account Credit</h4>
+							<span class="sr-only">{{ toFriendlyDateVerbose tuition_due }}</span>
 			        &nbsp;
 	                        <span class="card-badge-action pull-right">in {{ tuition_date_offset }} days</span>
 	                    </div>
 	                </div>
-		    </li>
-	            {{/if}}
-	        </ul>
-	    </div>
+	                <div class="cleafix middle-row">
+				    	<div class="pull-right" >
+				    		<div class="btn btn-default card-action-btn btn-rt"><a href="http://f2.washington.edu/fm/sfs/tuition/payment" target="_blank" data-linklabel="Make Tuition Payment">Make payment</a></div>
+						</div>
+		    		</div>
+				</li>
+	            {{ /if }}
+
         <!-- end if tuition_credit -->
-        {{/if}}
+        {{ /if }}
         <!-- end if has_balance -->
-    {{/if}}
-
-    {{#if pce_accbalance}}
-        <div class="card-badge-container">
-	    <ul class="card_list">
-		<li>
-	            <div class="card-badge clearfix">
-	                <div class="pull-left">
-	                    <h4 class="card-badge-label">You owe PCE</h4>
-	                </div>
-			<div class="pull-right">
-	                    <span class="card-badge-value text-danger">${{ pce_accbalance }}</span>
-			    &nbsp;
-	                    <span class="card-badge-action pull-right"><a href="https://www.pce.uw.edu/help/registration-costs/payment-options" target="_blank" data-linklabel="PCE Payment Options">Make payment</a></span><br/>
-                        <span class="pull-right" style="width: 100%; line-height: 1.1em; font-size: .9em; text-align: right; margin-top: -.2em;"> Contact PCE for due date</span>
-	                </div>
-	            </div>
-		</li>
-	    </ul>
-	</div>
-    {{/if}}
+    {{ /if }}
 
+    {{ #if pce_accbalance }}
+			<div class="card-badge-container ">
+	        <ul class="card_list">
+				<li class="pce-tuition">
+			        <div class="card-badge clearfix">
+			                <div class="pull-left">
+			                    <h4 class="card-badge-label">You owe PCE</h4>
+			                </div>
+							<div class="pull-right">
+			                    <span class="card-badge-value text-danger">${{ pce_accbalance }}</span>
+			                    <span class="pull-right" style="width: 100%; line-height: 2em; font-size: .9em; text-align: right;"><a href="https://www.pce.uw.edu/contact-us" target="_blank" title="PCE Help center contact information">Contact PCE for due date</a></span>
+			        		</div>            
+			        </div>
+			        <div class="clearfix middle-row">            
+			        	<div class="pull-right"><div class="btn btn-default card-action-btn btn-rt"><a href="https://www.pce.uw.edu/help/registration-costs/payment-options" target="_blank" data-linklabel="PCE Payment Options">Make payment to PCE</a></div>
+			        	</div>
+			        </div>
+				</li>
+    {{ /if }}
+			</ul>
+		    </div><!-- .card-badge-container tuition -->
```

Comment:
Should review this logic too. It looks like `{{ #if pce_accbalance }}` closes on line 115, but then the </ul> and </div> come after that. 

this means that if `{{ #if pce_accbalance }}` is false, you'll get an extra `</ul></div>` thrown in there.